pkg/3scale/amp/component/apicast_options_builder.go

```diff
@@ -9,12 +9,14 @@ import (
 
 type ApicastOptions struct {
 	// required options
-	appLabel       string
-	managementAPI  string
-	openSSLVerify  string
-	responseCodes  string
-	tenantName     string
-	wildcardDomain string
+	appLabel           string
+	managementAPI      string
+	openSSLVerify      string
+	responseCodes      string
+	tenantName         string
+	wildcardDomain     string
+	productionImageTag string
```

Comment:
why `productionImageTag` and `stagingImageTag`. Is there any use case where they will be different?

why not use `ampRelease` property like in other option structures

Reply:
The name/idea of "ampRelease" does not convince me in this cases. image tags do not necessarily have to have the same value as ampRelease so I wanted to diverge from that concept. I prefer to just provide the image tag that we want.
Both production and apicast images are the same. I think it's not a bad idea to be able to set them separately and the implementation. It's added flexibility for almost negligible implementation cost.

Reply:
There is no use case for having different tags (image references) for staging and production. They should be the same tag always as production gateway is expected to be the same image as staging, but different configuration.